controllers/cloud.redhat.com/providers/cronjob/impl.go

```diff
@@ -166,6 +166,10 @@ func applyCronCronJob(app *crd.ClowdApp, env *crd.ClowdEnvironment, cj *batch.Cr
 	labels["pod"] = nn.Name
 	app.SetObjectMeta(cj, crd.Name(nn.Name), crd.Labels(labels))
 
+	// add kubelinter annotations to ignore liveness/readiness probes on CronJobs
+	pt.ObjectMeta.Annotations["ignore-check.kube-linter.io/no-liveness-probe"] = "probes not required on Job pods"
+	pt.ObjectMeta.Annotations["ignore-check.kube-linter.io/no-readiness-probe"] = "probes not required on Job pods"
```

Comment:
can you change it to "probes not required for CronJob pods" ? just to please my ADHD :)

Reply:
I used this wording because technically a CronJob creates a Job :P